docs/topics/eclipse-about-plugin.adoc

```diff
@@ -9,11 +9,3 @@
 The {ProductName} ({ProductShortName}) plugin for Eclipse and Red Hat CodeReady Studio helps you migrate and modernize applications.
```

Comment:
Global. I really think you need to bring this into line with our conventions for all the modules: Migrate and modernize your applications by using the MTA plugin for X.